src/runtime/vdom/set-accessor.ts

```diff
@@ -109,11 +109,23 @@ export const setAccessor = (
         // except for the first character, we keep the event name case
         memberName = ln[2] + memberName.slice(3);
       }
-      if (oldValue) {
-        plt.rel(elm, memberName, oldValue, false);
-      }
-      if (newValue) {
-        plt.ael(elm, memberName, newValue, false);
+      if (oldValue || newValue) {
+        // Need to account for "capture" events.
+        // If the event name ends with "Capture", we'll update the name to remove
+        // the "Capture" suffix and make sure the event listener is setup to handle the capture event.
+        let capture = false;
+        if (memberName.endsWith(CAPTURE_EVENT_SUFFIX)) {
+          capture = true;
```

Comment:
Minor, couldn't we avoid the double assignment here of `capture` by using the conditional directly since we're always calling it?
```suggestion
        const capture = memberName.endsWith(CAPTURE_EVENT_SUFFIX);
        if (capture) {
```

Reply:
Made that change, also just removed the conditional that would follow so we always just make the call to `replace`, just won't do anything if there's nothing to replace (obviously).